connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TransformationChain.java

```diff
@@ -68,4 +74,11 @@ public int hashCode() {
         return Objects.hash(transformations);
     }
 
+    public String toString() {
+        StringJoiner chain = new StringJoiner(", ", "<" + getClass().getSimpleName() + ": ", ">");
```

Comment:
i think the same comment about `getSimpleName` applies here (and in the body of this method) as well